op-conductor/conductor/flashblocks_handler.go

```diff
@@ -0,0 +1,209 @@
+package conductor
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// StartFlashblocksHandler initializes the flashblocks handler and starts the rollup boost listener
+func (oc *OpConductor) StartFlashblocksHandler(ctx context.Context) error {
+	// Start the WebSocket server
+	if err := oc.startWebSocketServer(ctx); err != nil {
```

Comment:
If we don't configure a `RollupBoostWsURL`, I'm thinking we shouldn't start the websocket server, wdyt?